charts/prometheus/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: prometheus
 appVersion: v2.46.0
-version: 24.3.0
+version: 24.3.1
```

Comment:
Thank you, @starlightromero, for your PR.

As presented, it would introduce an incompatible change and would therefore require bumping the major number. I would suggest that we keep the toggle avoiding thus a breaking change.

Reply:
Good call out! Thanks @zeritti . If there is a way of queueing updates for the next major release, I suggest the code from the original PR be added. This would make the template similar to that of `kube-state-metrics`. I suggest this because it seems easier to maintain if all charts in this repo use a similar pattern